CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### v2.4.0 (Jun 13, 2019)
```

Comment:
Can you add an entry for `v2.3.0` also? just say that it is a duplicate of `v2.4.0`?